feature/qos/ate_tests/bursty_traffic_test/bursty_traffic_test.go

```diff
@@ -488,6 +488,14 @@ func ConfigureDUTIntf(t *testing.T, dut *ondatra.DUTDevice) {
 		a := s.GetOrCreateAddress(intf.ipAddr)
 		a.PrefixLength = ygot.Uint8(intf.prefixLen)
 		gnmi.Replace(t, dut, gnmi.OC().Interface(intf.intfName).Config(), i)
+		if deviations.ExplicitInterfaceInDefaultVRF(dut) {
```

Comment:
ate testcase was deprecated and removed as per https://github.com/openconfig/featureprofiles/pull/2114 .

Hence, we no longer need this change for ATE

Reply:
Fixed branch conflict with removal of ATE changes.